src/epcpm/cli/sunspectostaticmodbus.py

```diff
@@ -0,0 +1,343 @@
+import click
```

Comment:
`sunspec to staticmodbus` .. should this be controls by versions it supports ? ... for example

`sunspec 1.0 to staticmodbus 4.2`

Reply:
This script will run only two times at most. After that, it will be deprecated. No need to add logic for versions.